.changeset/common-candies-kiss.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': minor
+'example-app-next': minor
```

Comment:
This package is not published so no need to include

Reply:
Ok I removed it.